web/cobrands/peterborough/assets.js

```diff
@@ -163,42 +189,62 @@ fixmystreet.assets.add(light_defaults, {
     asset_item_message: ''
 });
 
-var bin_defaults = $.extend(true, {}, defaults, {
-    class: OpenLayers.Layer.PeterboroughVectorAsset,
-    select_action: true,
-    actions: {
-        asset_found: fixmystreet.message_controller.asset_found,
-        asset_not_found: fixmystreet.message_controller.asset_not_found
-    },
-    attributes: {
-        asset_details: function() {
-            var a = this.attributes;
-            return a.Reference + ", " + a.Location;
-        },
-        central_asset_id: 'OBJECTID'
-    },
-    asset_id_field: 'OBJECTID',
-    asset_type: 'spot'
-});
+var url_base = 'https://tilma.mysociety.org/resource-proxy/proxy.php?https://peterborough.assets/';
 
-fixmystreet.assets.add(bin_defaults, {
+var flytipping_defaults = $.extend(true, {}, arcgis_defaults, {
     http_options: {
       params: {
-        TYPENAME: 'LitterBins'
+        outFields: '',
       }
     },
-    asset_category: 'Litter bin',
-    asset_item: 'litter bin'
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    asset_category: ['General fly tipping', 'Hazardous fly tipping', 'Offensive graffiti', 'Non offensive graffiti'  ],
+    non_interactive: true,
+    road: true,
+    asset_item: 'road',
+    asset_type: 'road',
 });
 
-fixmystreet.assets.add(bin_defaults, {
+// PCC Property Combined
+fixmystreet.assets.add(flytipping_defaults, {
     http_options: {
-      params: {
-        TYPENAME: 'DogBins'
```

Comment:
Just to note to check whether this PR should be removing the dog/litter bin asset layers.

Reply:
My understanding was these weren't being used.